package.json

```diff
@@ -116,11 +117,17 @@
       "default": true,
       "order": 5
     },
+    "enableDicy": {
+      "description": "Enable the usage of the [DiCy](https://yitzchak.github.io/dicy/) builder instead of `latexmk`.",
+      "type": "boolean",
```

Comment:
My point was that if we think we'll migrate entirely to DiCy at some point not too far into the future, it is more user friendly to have an experimental "Enable DiCy" setting that will get removed once we migrate. If we decide to keep both latexmk and DiCy around for a while, down the line, we can always re-introduce the "Builder" enumeration setting then.

Unless you disagree, or have a different viewpoint, I think we should stick with what you originally had here. Also think we should call it "Use DiCy builder" or some such, and it might make sense to point out that this is an experimental feature for the time being. Thoughts?